NHSD.BuyingCatalogue.Ordering.Api.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
Missing the solution folder labelled 'src'. This should contain the API project. So will need the same folder on disk as well. This is just for consistency purposes and believe has worked well in the past.

Have a look at one of the other repositories e.g. [Buying Catalogue Identity](https://github.com/nhs-digital-gp-it-futures/BuyingCatalogueIdentity)